### DISCUSSION TITLE
Cross section values for 13.6 TeV

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/nanoAOD_vvVBS#45] [Show and tell]
Xs values

**Comment by YujiLee301** [comment]
DYTo2L_MLL-50_TuneCP5_13p6TeV_pythia8 5556.0
ZZto4L_TuneCP5_13p6TeV_powheg-pythia8 1.39
WZto3LNu_TuneCP5_13p6TeV_powheg-pythia8 4.924
TT_TuneCP5_13p6TeV_powheg-pythia8 762.3

**Comment by ram1123** [comment]
Updated cross-section values

Reference: [SMP-24-001](https://cms.cern.ch/iCMS/analysisadmin/cadilines?line=SMP-24-001&tp=an&id=2758&ancode=SMP-24-001)

<img width="528" alt="Screenshot 2024-09-03 at 11 42 59" src="https://github.com/user-attachments/assets/47c3a990-4ece-456b-a14d-899ac1dc97cd">

**Comment by ram1123** [comment]
# LHCHXSWG cross-section for higgs boson production at 13.6 TeV

https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWG136TeVxsec_extrap
